src/components/side-suggested-users.js

```diff
@@ -71,13 +100,18 @@ export default class SideSuggestedUsers extends React.Component {
               />
             </div>
 
-            <div className="layout__row layout__row-small">
+            <div className="layout__row layout__grid layout__row-small">
               <FollowButton
                 active_user={current_user}
                 following={i_am_following}
                 triggers={triggers}
                 user={user}
               />
+              <IgnoreButton
+                active_user={current_user}
+                ignoreUser={this.ignoreUser}
+                user={user}
```

Comment:
property-names are ambiguous. all 3 look like some kind of user-objects, but second one is a callback. how about `onClick`?